lwesp/src/system/lwesp_ll_template.c

```diff
@@ -89,7 +99,7 @@ lwesp_ll_init(lwesp_ll_t* ll) {
     }
 
     /* Step 3: Configure AT port to be able to send/receive data to/from ESP device */
-    configure_uart(->uart.baudrate);            /* Initialize UART for communication */
```

Comment:
Why written as "`->uart.baudrate`" instead of "`ll->uart.baudrate`".

Reply:
Plus there's missing function definition` configure_uart`. At least a function body should be provided.